code/modules/uplink/uplink_items.dm

```diff
@@ -2297,3 +2300,420 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
+	item = /obj/item/ammo_box/magazine/m556
+	cost = 4
+
+/datum/uplink_item/nt/ammo/tommyammo
+	name = ".45 Drum Magazine"
+	desc = "An additional 50-round .45 drum magazine; suitable for use with the Thompson SMG."
+	item = /obj/item/ammo_box/magazine/tommygunm45
+	cost = 4
+
+/datum/uplink_item/nt/ammo/m45ammo
+	name = ".45 Handgun Magazine"
+	desc = "An additional 8-round .45 magazine; suitable for use with the M1911."
+	item = /obj/item/ammo_box/magazine/m45
+	cost = 1
+
+/datum/uplink_item/nt/ammo/saberammo
+	name = "9mm Magazine"
+	desc = "An additional 21-round 9mm magazine; suitable for use with the Saber SMG."
+	item = /obj/item/ammo_box/magazine/smgm9mm
+	cost = 1
+
+/datum/uplink_item/nt/ammo/saberammoap
+	name = "9mm AP Magazine"
+	desc = "An additional 21-round 9mm magazine with armor piercing rounds; suitable for use with the Saber SMG."
+	item = /obj/item/ammo_box/magazine/smgm9mm/ap
+	cost = 2
+
+/datum/uplink_item/nt/ammo/saberammoin
+	name = "9mm Incendiary Magazine"
+	desc = "An additional 21-round 9mm magazine loaded loaded with incendiary rounds; suitable for use with the Saber SMG."
+	item = /obj/item/ammo_box/magazine/smgm9mm/fire
+	cost = 1
+
+/datum/uplink_item/nt/ammo/wt
+	name = "4.6x30mm Magazine"
+	desc = "An additional 22-round 4.6x30mm magazine; suitable for use with the WT-550."
+	item = /obj/item/ammo_box/magazine/wt550m9
+	cost = 1
+
+/datum/uplink_item/nt/ammo/wtap
+	name = "4.6x30mm AP Magazine"
+	desc = "An additional 22-round 4.6x30mm magazine loaded with armor piercing rounds; suitable for use with the WT-550."
+	item = /obj/item/ammo_box/magazine/wt550m9/wtap
+	cost = 2
+
+/datum/uplink_item/nt/ammo/wtic
+	name = "4.6x30mm Incendiary Magazine"
+	desc = "An additional 22-round 4.6x30mm magazine loaded with incendiary rounds; suitable for use with the WT-550."
+	item = /obj/item/ammo_box/magazine/wt550m9/wtic
+	cost = 2
+
+/datum/uplink_item/nt/ammo/wtr
+	name = "4.6x30mm Rubber Shot Magazine"
+	desc = "An additional 22-round 4.6x30mm magazine loaded with less lethal rounds; suitable for use with the WT-550."
+	item = /obj/item/ammo_box/magazine/wt550m9/wtr
+	cost = 1
+
+/datum/uplink_item/nt/cqc
+	category = "Close Quarters Combat"
+
+/datum/uplink_item/nt/cqc/esword
+	name = "Energy Sword"
+	desc = "The energy sword is an edged weapon with a blade of pure energy. The sword is small enough to be \
+			pocketed when inactive."
+	item = /obj/item/melee/transforming/energy/sword/saber
+	cost = 8
+
+/datum/uplink_item/nt/cqc/eshield
+	name = "Energy Shield"
+	desc = "A shield that blocks all energy projectiles but is useless against physical attacks."
+	item = /obj/item/shield/energy
+	cost = 16
+
+/datum/uplink_item/nt/cqc/ntmantisblade
+	name = "H.E.P.H.A.E.S.T.U.S. Mantis Blades"
+	desc = "A pair of retractable arm-blade implants. Activating both will let you double-attack."
+	item = /obj/item/storage/briefcase/nt_mantis
+	cost = 10
```

Comment:
Let's realistically consider what advantages this has over the e-sword.

The e-sword has effectively 6 less damage (since HEP-mantis double attacks with 18 brute), but the 50% block chance and 35 AP (?). Downside is that it's an item you have to pull out (tiny btw) and activate, meaning it can be disarmed.

Block chance and AP mean it's kinda harsh for something that occupies both of your arms when you can just dip an additional 3 TC for something that WILL let you insta-win against most combatants, which makes you just straight up better at dueling.

Personally I'd just buff both HEP and GOR mantis blades, especially the former, since the hidden implant is a lot more important for a traitor than an ERT.

Reply:
^ 🤓

Reply:
36 damage

Reply:
Assuming no other modifiers that is one less hit on a non-armored target. Issue is is that ERT will probably be dealing with armored targets.

My inclination is 6-7 but that feels far too low, I absolutely believe it's not as good as the esword but does hefty enough damage to make it around that 8 cost range.

I might make it meatier but far pricier in a later PR because after I did basic-ass math I better realized why GORLEX blades are reasonably potent.

Reply:
ill change it to 7 and we'll go from there